src/stores/rootReducer.ts

```diff
@@ -6,14 +6,16 @@ import MetaReducer from './meta/MetaReducer';
 import {reducer as formReducer} from 'redux-form';
 import RenderReducer from './render/RenderReducer';
 import IStore from './IStore';
+import ModalReducer from './modal/ModalReducer';
 
 const reducers: IStore = {
     form: formReducer,
     loadingReducer: LoadingReducer.reduce as any,
     metaReducer: MetaReducer.reduce as any,
+    modalReducer: ModalReducer.reduce as any,
```

Comment:
why all the casting here?

Reply:
Not sure of the best way to do this so I put any just to get it to work.